react/Header/Navigation/Navigation.js

```diff
@@ -51,24 +51,28 @@ export default function Navigation({ locale, linkRenderer, activeTab, newBadgeTa
 
       <ul className={styles.list} data-automation="nav-tabs">
         {
-          items.map(({ specificLocale = locale, analytics, name, isShort, ...restProps }, i) => {
-            const badgeStyles = classnames({
-              [styles.newBadge]: true,
-              [styles.newBadge_isShort]: isShort
-            });
-
+          items.map(({ specificLocale = locale, analytics, name, isShort, ...restProps }, key) => {
             return (
               (specificLocale === locale) ?
-                <li className={styles.item} key={i}>
+                <li className={styles.item} key={name}>
                   {
                     linkRenderer({
                       children: [
                         name,
                         name === newBadgeTab && (
-                          <NewBadge
+                          <span
                             key={name}
-                            className={badgeStyles}
-                          />
+                            className={classnames({
+                              [styles.newBadge]: true,
+                              [styles.newBadge_isShort]: isShort,
+                              [styles.newBadge_isLast]: items.length - 1 === key
+                            })}>&nbsp;
+                            <Badge
```

Comment:
Any reason you've gone with `<Badge children="New" />` over `<Badge>New</Badge>`? I think the rest of the styleguide uses the later so perhaps that would keep things more consistent?

Reply:
Don't see any difference really...